merge_data/shop_FR.mapping.json

```diff
@@ -1016,5 +1227,28 @@
     "generate": {
       "shop": "funeral_directors"
     }
+  },
+  {
+    "missing_official": 8310,
+    "missing_osm": 7210,
+    "class": "96.04Z",
+    "level": 3,
+    "title": "Entretien corporel",
+    "match": [
+      {
+        "shop": "beauty"
+      },
+      {
+        "amenity": "swingerclub"
+      },
+      {
+        "shop": "massage"
+      }
+    ],
+    "generate":
+    {
+      "shop": "beauty;massage",
+      "fixme": "Select an amenity type. Use amenity=swingerclub for swinger clubs (club libertin)"
```

Comment:
Do you thin it is generic to use 96.04Z for swingerclub?

Reply:
Yes I was quite surprised when testing 96.04Z in Lyon to find out that all swingerclubs in Lyon are categorised 96.04Z. I just checked two also in Bordeaux and Paris and it seems to be the same.

Reply:
Ok, thank.